.github/workflows/pysa.yml

```
@@ -0,0 +1,53 @@
+name: pysa
+
+on:
+  push:
+    branches: [ test-2, master ]
+  pull_request:
+    branches: [ master ]
+
+  workflow_dispatch:
+
+jobs:
+  pysa:
+    runs-on: ubuntu-latest
+
+    steps:
+      - uses: actions/checkout@v2
+      - uses: actions/setup-python@v2
+
+      - name: Install Dependencies
+        run: |
+          python -m pip install -r requirements.txt
+          sudo apt-get install ocaml
+          sudo apt-get install ocaml-dune
```

nitpick: merging those 2 lines would be faster
```suggestion
          sudo apt-get install ocaml ocaml-dune
```